index.html

```diff
@@ -3213,6 +3213,28 @@ <h2>
           <li>Set <var>response</var>.<a>[[\retryPromise]]</a> to
           <var>retryPromise</var>.
           </li>
+          <li data-link-for="PaymentValidationErrors">If
+          <var>errorFields</var>'s <a>paymentMethod</a> member was passed, and
+          if required by the specification that defines <var>response</var>'s
+          <a>payment method</a>, then <a data-cite=
+          "!WEBIDL#dfn-convert-ecmascript-to-idl-value">convert</a>
+          <var>errorFields</var> <a>paymentMethod</a> to an IDL value.
+          Otherwise, <a data-cite=
```

Comment:
Unsure whether this buys anything, versus just ignoring it. Maybe it's useful for web-based payment handlers??

Reply:
Yeah, that's what I was thinking... not all payment handlers will be formally specified, so this allows stuff to be passed to "https://bob.pay".